function.js

```diff
@@ -0,0 +1,129 @@
+document.addEventListener("DOMContentLoaded", () => {
+   let pokemon1 = document.querySelector("#pokemon1")
+   let pokemon2 = document.querySelector("#pokemon2")
+   let getPoke = document.querySelector("#getPoke")
+   let battle = document.querySelector("#battle")
+   let pokemon = ""
+   let pokemonchar = ""
+   let poke1 = document.createElement("img")
+   let poke2 = document.createElement("img")
+   let status1 = ""
+   let status2 = ""
+   let mymove1 = ""
+   let mymove2 = ""
+   let h1 =document.createElement("h1")
+   let h2 =document.createElement("h1")
+   let p1 =document.createElement("p")
+   let p2 =document.createElement("p")
+   let movelistUl = document.createElement("ul")
+   let movelistUl2 = document.createElement("ul")
+   let hp1 = document.createElement("h5")
+   let hp2 = document.createElement("h5")
+      
+   const randomNumber = (max, min) => Math.floor(Math.random() * (max -min)) + min
+
+
+
+    const fetchData = async () =>{
```

Comment:
Having an argument here to append to a specific html tag would decrease the amount of code you have here by half